### PR TITLE
WS-1741: add new Configuration request support for dynamic gazetteer

### PR DESCRIPTION
**Pull request by Katsuya-Tomioka**

### CODE REVIEW

**Comment by jkho** [comment]
Please merge master into this branch

**Comment by Katsuya-Tomioka** [comment]
Done.

File: model/src/main/java/com/basistech/rosette/apimodel/ConfigurationResponse.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * Response ping data
+ */
+@Getter
+@EqualsAndHashCode
+@Builder
+@JacksonMixin
+public class ConfigurationResponse extends Response {
+    /**
+     * @return the ping response message
```
Comment (jkho) [inline model/src/main/java/com/basistech/rosette/apimodel/ConfigurationResponse.java]: Update the comment. Not ping response.

File: model/src/main/java/com/basistech/rosette/apimodel/ConfigurationResponse.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+package com.basistech.rosette.apimodel;
+
+import com.basistech.rosette.annotations.JacksonMixin;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+/**
+ * Response ping data
```
Comment (jkho) [inline model/src/main/java/com/basistech/rosette/apimodel/ConfigurationResponse.java]: Not ping data

Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/ConfigurationResponse.java]: fixed.

File: model/src/main/java/com/basistech/rosette/apimodel/ConfigurationResponse.java (hunk 1 above)
Comment (Katsuya-Tomioka) [reply model/src/main/java/com/basistech/rosette/apimodel/ConfigurationResponse.java]: done.

**Pull request merged**